packages/stream/common/types.ts

```diff
@@ -83,6 +87,11 @@ export interface ITransactionResult {
   txId: string;
 }
 
+export interface IFees {
+  streamflow_fee: number;
```

Comment:
Let's follow the most commonly used camelCase here since other interfaces don't have snake_case as well.